<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>devicePlannerForm</class>
 <widget class="QWidget" name="devicePlannerForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="spacing">
    <number>20</number>
   </property>
   <property name="leftMargin">
    <number>15</number>
   </property>
   <property name="bottomMargin">
    <number>15</number>
   </property>
   <item>
    <widget class="QLabel" name="m_labelTitle">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Device Planner</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="m_labelDetail">
     <property name="text">
      <string>Select the series and device you want to target for compilation.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="20,10,70">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>30</number>
     </property>
     <property name="bottomMargin">
      <number>40</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Filters</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="30,30,30">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,90">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Family:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,90">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="m_labelSeries">
            <property name="text">
             <string>Series:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_comboBoxSeries"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,90">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="m_labelPackage">
            <property name="text">
             <string>Package:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="m_comboBoxPackage"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Available devices:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_groupBoxGrid">
       <property name="title">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
